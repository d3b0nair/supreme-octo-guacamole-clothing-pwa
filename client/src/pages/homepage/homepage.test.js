import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage.component";

describe("Home Page", () => {
  it("should render HomePage component", () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
