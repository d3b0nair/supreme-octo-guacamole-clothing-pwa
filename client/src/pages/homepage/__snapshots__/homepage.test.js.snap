// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render HomePage component 1`] = `
<styled.div>
  <Connect(Directory) />
</styled.div>
`;
